packages/@expo/fingerprint/README.md

```diff
@@ -127,9 +154,23 @@ console.log(result);
 
 `npx @expo/fingerprint /path/to/projectRoot fingerprint.json`
 
+## Customizations
+
+### Include or exclude extra files in the **.fingerprintignore** file
+
+The fingerprint library has a default [`DEFAULT_IGNORE_PATHS`](https://github.com/expo/expo/blob/main/packages/%40expo/fingerprint/src/Options.ts#L9) to keep hashing speed and fingerprint result stability. If the default setup does not fit your workflow, you can either include or exclude files in a **.fingerprintignore** file inside your project root. The syntax of **.fingerprintignore** is similar to [.gitignore](https://git-scm.com/docs/gitignore#_pattern_format).
```

Comment:
```suggestion
The default paths [`DEFAULT_IGNORE_PATHS`](https://github.com/expo/expo/blob/main/packages/%40expo/fingerprint/src/Options.ts#L9) are designed for speed of hashing and for fingerprint result stability. If the default setup does not fit your workflow, you can either include or exclude files in a **.fingerprintignore** file inside your project root. The syntax of **.fingerprintignore** is similar to [**.gitignore**](https://git-scm.com/docs/gitignore#_pattern_format).
```

Reply:
are there any differences between the fingerprintignore and gitignore syntax that we should point out?